support/pom.xml

```diff
@@ -15,30 +15,9 @@
   <!-- This is an incubator for code we expect to get hosted by target plugins -->
 
   <properties>
-    <useBeta>true</useBeta>
-    <access-modifier-checker.skip>true</access-modifier-checker.skip>
-    <rootdir>${basedir}/..</rootdir>
+    <hpi.compatibleSinceVersion>1.18</hpi.compatibleSinceVersion>
```

Comment:
does this look right to you @oleg-nenashev or do you want it on the main plugin?

Reply:
WDYT @timja

Reply:
I think `support` plugin is better. Otherwise the warning will popup for users who do not do credentials/RoleBased configurations and hence do not need this plugin

Reply:
Cool 😅